src/example-report.html

```diff
@@ -56,7 +56,7 @@ <h1>Abstract</h1>
                   Generalisations</em>. Each major section lists model items that need correction of errors
```

Comment:
1. please move the example report into the /data folder

2. please sort each of the nomenclature lists alphabetically.

Reply:
done